Adapted README.md reflecting how released buildpack can be used (#13)
Dynatrace/heroku-buildpack-dynatrace: README.md

To integrate Dynatrace OneAgent into your existing project you need to add the Dynatrace buildpack to your project's buildpacks and set your Dynatrace environment ID and token. For complete details, please see the [Dynatrace Heroku installation guidelines](https://www.dynatrace.com/support/help/cloud-platforms/heroku/how-do-i-monitor-heroku-applications/).

```shell

> # Highly recommended to add the latest released version of the Dynatrace heroku buildpack:

Reviewer: ```suggestion
# Add a released version of the Dynatrace Heroku buildpack:
```

Reviewer: Add a released version of the Dynatrace Heroku buildpack
